skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +335,92 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom_basic(self):
```

Comment:
Awesome, thanks!!

Sorry for so many comments on your PR. I've implemented these types of methods before and there are a surprising number of border cases and gotchas that are easy to miss and have bit me (or worse, users) down the road so I'm trying to hammer this out now. Thanks for your patience and for making all these changes!